web/modules/custom/grundsalg_db_client/src/Controller/ApiController.php

```diff
@@ -7,13 +7,57 @@
 namespace Drupal\grundsalg_db_client\Controller;
 
 use Drupal\Core\Controller\ControllerBase;
-use GuzzleHttp\Client;
+use Drupal\Core\File\FileSystemInterface;
+use Drupal\Core\File\FileUrlGeneratorInterface;
+use Drupal\Core\Http\RequestStack;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\grundsalg_db_client\Service\ContentCreationService;
+use Drupal\itkore_admin\State\BaseConfig;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 /**
  * ApiController.
  */
-class ApiController extends ControllerBase {
+class ApiController extends ControllerBase
+{
+  private RequestStack $requestStack;
+  private BaseConfig $config;
+  private ContentCreationService $contentCreationService;
+  private FileUrlGeneratorInterface $fileUrlGenerator;
+  private FileSystemInterface $fileSystem;
+
+  public const PUBLIC_BASE_PATH = 'public://api/udstykning';
```

Comment:
Right now the path is `public/api/udstykning` e.g https://fagsystem.grundsalgaarhus.dk/public/api/udstykning/301/grunde

Reply:
Let us configure the Angular-widget to use the new `/sites/default/files/` path